reviewing/list/src/index.ts

```diff
@@ -7,6 +7,7 @@ import { renderJson } from './renderJson'
 import { renderParseable } from './renderParseable'
 import { renderTree } from './renderTree'
 import { type PackageDependencyHierarchy } from './types'
+import { pruneTreeToGetFirst10EndLeafs } from './pruneTree'
```

Comment:
Instead of hardcoding 10 into the name of the function, just pass the limit as an option.